src/h5web/visualizer/Visualizer.test.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { fireEvent, render, screen } from '@testing-library/react';
-import { mockValues } from '../../packages/lib';
+import { mockValues } from '../providers/mock/data';
```

Comment:
VSCode has been annoying me by auto-importing from `/packages` a lot of the time. 💢